master/internal/command/authz_tensorboard_iface.go

```diff
@@ -0,0 +1,36 @@
+package command
+
+import (
+	"context"
+
+	"github.com/determined-ai/determined/proto/pkg/tensorboardv1"
+
+	"github.com/determined-ai/determined/master/internal/authz"
+	"github.com/determined-ai/determined/master/pkg/model"
+)
+
+// TensorboardAuthZ describes authz methods for Tensorboards.
+type TensorboardAuthZ interface {
+	// GET /api/v1/tensorboards/:tb_id
+	CanGetTensorboard(
+		ctx context.Context, curUser *model.User, tb *tensorboardv1.Tensorboard,
+	) (canGetTensorboard bool, serverError error)
```

Comment:
Right sorry I forgot to follow up on that. I checked and I didn't see any real reason to keep it, I was going to check with rbac team.

Reply:
consulted @maxrussell about this. TLDR is that our underlying implementation just uses a special error so we _could_ simplify the interface, but from an api design standpoint it's a good idea to be clear what is a permission granted/denied and what is a server error. From a golang standpoint, this makes sense and is pretty typical, such as having something like `ok, err := fn()`.

Reply:
> It's a good idea to be clear what is a permission granted/denied and what is a server error.

I think we definitely want regardless of how we get there.

> From a golang standpoint, this makes sense 

by "this" do you mean `bool, err` return type instead of `permErr, serverErr error` ? or the other way